tests/foreman/api/test_config_template.py

```diff
@@ -30,56 +30,36 @@ def test_build_pxe_default(self):
         response.raise_for_status()
         self.assertIsInstance(response.json(), dict)
 
-    def test_associate_config_template_with_organization(self):
-        """@Test: Associate config template with organization
+    def test_config_template_orgs(self):
+        """@Test: Associate a config template with organizations.
 
         @Assert: Config template is associated with organization
 
         @Feature: ConfigTemplate
 
         """
-        org_1 = entities.Organization().create()
-        org_2 = entities.Organization().create()
+        orgs = [entities.Organization().create() for _ in range(2)]
 
         # By default, a configuration template should have no organizations.
         conf_templ = entities.ConfigTemplate().create()
         self.assertEqual(0, len(conf_templ.organization))
 
         # Associate our configuration template with one organization.
-        client.put(
-            conf_templ.path(),
-            {'config_template': {'organization_ids': [org_1.id]}},
-            verify=False,
-            auth=get_server_credentials(),
-        ).raise_for_status()
-
-        # Verify that the association succeeded.
-        conf_templ = conf_templ.read()
-        self.assertEqual(1, len(conf_templ.organization))
-        self.assertEqual(org_1.id, conf_templ.organization[0].id)
+        conf_templ.organization = orgs[:1]
+        conf_templ = conf_templ.update(['organization'])
```

Comment:
Now, we *could* do something like automatically issue an HTTP PUT request to the server as soon as the `.organization` attribute is assigned to. But keeping local objects in sync with the server automatically is a nightmarishly hard thing to do. (How do you register your client as a listener for events on the server? How do you deal with editing conflicts? What happens if you miss an event? Is this single-threaded?) And it's totally unnecessary, IMO.